tools/ci_build/github/azure-pipelines/bigmodels-ci-pipeline.yml

```diff
@@ -293,7 +293,7 @@ stages:
 
     - template: templates/get-docker-image-steps.yml
       parameters:
-        Dockerfile: tools/ci_build/github/linux/docker/Dockerfile.package_ubi8_cuda11_8_tensorrt8_6
+        Dockerfile: tools/ci_build/github/linux/docker/Dockerfile.package_ubi8_cuda11_8_tensorrt10_0
```

Comment:
this doesn't match the name of the file.

Reply:
this has been fixed